en/drafts/originals/clustering-visualizing-word-embeddings.md

```diff
@@ -357,25 +352,25 @@ tune_figure(axs[1], 'DDC2 Group')
+Of particular note in Figure 3 should be the areas where the DDC classification does *not* appear to align with the location of the thesis in the reduced corpus-space. With a bit of squinting you should be able to see a small number of Linguistics theses over by History of the Ancient World, and some History theses towards the 'far' side of the Philosophy grouping. We'll take a slightly closer look at these later, but any way you look at it this is a promising start. There are distinct groupings in the data that seem to map fairly well on to the classes assigned by experts. In short, it's not a jumble of overlapping DDCs!
 
 ## Hierarchical Clustering
 
 We could stop here and say "Well clearly we've done something useful", but it it would behoove us to cluster the data independently of its expert classification and compare the results in a more rigorous way. On both a practical and a philosophical level we feel that a [hierarchical clustering](https://docs.scipy.org/doc/scipy/reference/cluster.hierarchy.html) approach is more appropriate: all PhDs are, in some sense, related to one another as they must build on (and distinguish themselves from) what has come before, but we hope that you will also agree on the existence of at least *some* high-level distinctions between, say, the interests of Historians and Linguists, especially when it comes to how they write about their work!
 
 ### How it works
 
-Hierarchical clustering takes a 'bottom-up' approach: every document starts in its own cluster of size 1. Next, we merge the two *closest* 'clusters' in the data set to create a cluster of size 2. We then look for the *next closest* pair of clusters, and this search includes the centroid of the cluster of size 2 that we created in the preceding step. We progressively join documents to clusters and, ultimately, clusters to clusters, such that we end up with everything in one mega-cluster containing the entire corpus. This generates a tree of relationships that we can 'cut' at different levels: delving down branches in order to investigate relations at a finer scale but also able to see where and when clusters merged to form larger groups.
+Hierarchical clustering takes a 'bottom-up' approach. Every document starts in its own cluster of size 1. Next, we merge the two *closest* 'clusters' in the data set to create a cluster of size 2. We then look for the *next closest* pair of clusters, and this search includes the centroid of the cluster of size 2 that we created in the preceding step. We progressively join documents to clusters and, ultimately, clusters to clusters, such that we end up with everything in one mega-cluster containing the entire corpus. This generates a tree of relationships that we can 'cut' at different levels; delving down branches in order to investigate relations at a finer scale but also able to see where and when clusters merged to form larger groups.
```

Comment:
Size '1' and size '2' left as integers in this paragraph. This feels right as I think you are referring to the sizes given within your code? 

(Elsewhere, [our convention is](https://programminghistorian.org/en/author-guidelines#numbers) to write out numbers from one to nine and use integers above 10).